api/src/main/java/org/openmrs/Encounter.java

```diff
@@ -264,6 +284,25 @@ public void setLocation(Location location) {
 	public Set<Obs> getAllObs() {
 		return getAllObs(false);
 	}
+
+	/**
+	 * Returns all encounter obs as a flatten list(Set) of obs.
+	 * @param includeVoided indicates whether or not to include voided obs
+	 * @return a Set of all encounter' Obs
+	 */
```

Comment:
Can we add the since annotation? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NewPublicMethodsandClasses

Reply:
cool call @dkayiwa fyi @cioan